assets/js/modules/analytics-4/datastore/accounts.js

```diff
@@ -41,6 +43,35 @@ const baseInitialState = {
 };
 
 const baseActions = {
+	/**
+	 * Gets an ID of an account that has a matching property.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return {string} The account ID on success, otherwise NULL.
+	 */
+	*matchAccountID() {
+		const registry = yield commonActions.getRegistry();
+		const accounts = yield Data.commonActions.await(
+			registry.__experimentalResolveSelect( STORE_NAME ).getAccountSummaries()
+		);
+
+		if ( ! Array.isArray( accounts ) || accounts.length === 0 ) {
+			return null;
+		}
+
+		const url = registry.select( CORE_SITE ).getReferenceSiteURL();
+		const ga4PropertyIDs = ( Array.isArray( accounts ) ? accounts : [] ).reduce(
```

Comment:
The `Array.isArray` check is redundant here since we already have that above.
```suggestion
		const ga4PropertyIDs = accounts.reduce(
```